integration_tests/__tests__/override-globals.test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import runJest from '../runJest';
+
+test('overriding native promise does not freeze Jest', () => {
+  const run = runJest('override-globals');
+  expect(run.stderr).toMatch('PASS __tests__/index.js');
```

Comment:
You need to replace the slash for windows to be happy

Reply:
Yay, thanks! 🙂 Done